demoapp/malwarelytics.xml

```diff
@@ -63,7 +63,8 @@
             username="SERVICE_USERNAME_APPLE"
             password="SERVICE_PASSWORD_APPLE"
             publicKey="SERVICE_PUB_KEY_APPLE"
-        />
+        /> 
+
 
          <!-- 
             netrc file configuration (for private CocoaPods access)
```

Comment:
This is not a direct issue of the PR, but the `.netrc/_netrc file in your home folder.` sentence is highly misleading because it can be understood as: `/Users/myname/.netrc/_netrc`.

The `_netrc` is for Windows but iOS does not compile for Windows so this should probably not be mentioned.

Reply:
Removed `_netrc` mention